__fixtures__/helper.js

```diff
@@ -62,6 +62,9 @@ module.exports = {
           }
         },
         pullRequests: {
+          getFiles: () => {
+            return { data: options.files && options.files.map(file => ({filename: file, status: 'modified'})) }
```

Comment:
Or `deleted` actually...

Reply:
hmm.. why would `added` file be a dependent file? imagine this scenario 
lets say you have `user.js` and `avatar.js` as required dependent file and only `user.js` exists in the current repo. 
This doesn't make sense because you say two files are dependent but they are not even in the repo

Reply:
I can add it if you insist, its just that scenario shouldn't exist in the first place

Reply:
I'm not insisting. Was asking :-) Makes sense that it shouldn't bother with `added` 

'Though I think we should consider `deleted` is someone says that two files are dependent and the other is deleted that would be wrong. Thoughts?